addons/resource.language.en_gb/resources/strings.po

```diff
@@ -8640,8 +8655,10 @@ msgctxt "#19056"
 msgid "New timer"
 msgstr ""
 
+#. Label for a context menu entry to open timer settings dialog
+#: xbmc/pvr/windows/GUIWindowPVRTimers.cpp
 msgctxt "#19057"
-msgid "Edit timer"
+msgid "Settings"
```

Comment:
not sure about that one. At least to me it would not be obvious what "settings" means in this context, especially when in other places the "settings" context menu item brings you directly to related section of system settings (this might have been dropped already, not sure, but still). So existing label would be fine for me.